rpm/maep-qt.spec

```diff
@@ -69,6 +69,7 @@ install -m 644 -p /usr/lib/libfreetype.so.6 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicui18n.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicuuc.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicudata.so.52 %{buildroot}/usr/share/%{name}/lib/
+install -m 644 -p /lib/ld-linux.so.? %{buildroot}/usr/share/%{name}/lib/
 
```

Comment:
Yes, indeed. I compiled maep-qt for the Jolla tablet, which is based on the i486 architecture. Sailfish OS on i486 has `ld-linux.so.2` in `/lib`. Sailfish OS on ARM, however, has `ld-linux.so.3` (but not `ld-linux.so.2`), which is a link to `ld-linux-armhf.so.3`.